### ISSUE TITLE
Everything vanished

### ISSUE THREAD

**Issue by PongoSapiens** [raoulvdberge/refinedstorage#2119]
#### Issue description:
I was busy setting up auto crafting for some botania items, opened my crafting grid, and it was completely empty, no items, no craftables, nothing from external storage (water drum, lava drum, and an interface that keeps items in stock.
I have a large reborn storage multi block crafter that contained a lot of patterns, and 4 disks (2x16M,1x256M,1x64K) with approximately 1.5 million items stored.

I restarted the server, which didn't change anything.
I entered creative mode and created a new creative controller, disk drive, and grid; and tried moving the disks to the new system, but still nothing was found.

I've checked refinedstorage_disks.dat and everything seems to still be in there, so I'm not sure what to try next.

#### What happens:
Nothing can be accessed any more.
#### What you expected to happen:
Not to lose everything

#### Steps to reproduce:
I'm not aware of anything that I did that might of triggered this.

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge: 14.23.5.2768
- Refined Storage: 1.6.12
- Refined Storage Addons: 0.4.3
- Reborn Storage: 3.3.0.72
- Reborn Core: 3.10.5.350

We're using a modpack that I created Neopolitan v1.14

Does this issue occur on a server? [yes/no] yes

I have complete access to the server, and can provide whatever is needed to help diagnose this.

**Comment by PongoSapiens** [comment]
Update: I restored world/data/refinedstorage_nodes.dat from backup, and everything except the patterns in the multi-block crafter seems to be back. Now comparing the entries in the two refinedstorage_nodes.dat to see if I can identify a significant issue.

**Comment by PongoSapiens** [comment]
New Update:
This may be a reborn storage issue, with the system in it "broken" state, refined storage could see nothing through any of the grids, and the reborn storage multi block also claims to have no recipes.

I just resurrected the "broken" world, and started breaking the reborn storage multi block, and I was showered with patterns. Once I had completed the demolition, storing the patterns in chests, refined storage could suddenly see everything but the patterns that were in the multi-block.

Turning off the RS controller crashed my client, but everything *seems* to be working at the moment.
